ui/components/component-library/icon/icon.stories.tsx

```diff
@@ -74,37 +57,17 @@ export default {
     className: {
       control: 'text',
     },
-    marginTop: {
-      options: marginSizeControlOptions,
-      control: 'select',
-      table: { category: 'box props' },
-    },
-    marginRight: {
-      options: marginSizeControlOptions,
-      control: 'select',
-      table: { category: 'box props' },
-    },
-    marginBottom: {
-      options: marginSizeControlOptions,
-      control: 'select',
-      table: { category: 'box props' },
-    },
-    marginLeft: {
-      options: marginSizeControlOptions,
-      control: 'select',
-      table: { category: 'box props' },
-    },
```

Comment:
Shows up automatically now with the magic of TypeScript

<img width="1440" alt="Screenshot 2023-03-24 at 6 31 38 PM" src="https://user-images.githubusercontent.com/8112138/227678123-ca458de4-66c1-4471-a62e-032bb2e6b777.png">